NoSqlAssignment/blog_schema.doc

```diff
@@ -0,0 +1,88 @@
+
+Following schema is for collection posts:
+
+{
+	_id:post_id ,
+	post_heading : "Title of the post" ,
+	
+	post_body : "Contains description of post in form of text, images, html files etc" ,
+	
+	post_made_by : "Username of the post" ,
+	
+	post_link : "Contains post link to be shown" ,
```

Comment:
Why do you need to store post links int he db? This is something that should either get generated by the app, or embedded in the content.